src/main/java/seedu/address/logic/commands/DepleteIngredientCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INGREDIENT_QUANTITY;
+import static seedu.address.model.IngredientModel.PREDICATE_SHOW_ALL_INGREDIENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ingredient.Ingredient;
+import seedu.address.model.ingredient.IngredientQuantity;
+import seedu.address.model.person.exceptions.DuplicateItemException;
+
+/**
+ * Depletes the quantity of existing ingredient in the restaurant book.
+ */
+
+public class DepleteIngredientCommand extends Command {
+
+    public static final String COMMAND_WORD = "depleteingredient";
+    public static final String COMMAND_ALIAS = "dpi";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Depletes the quantity of the ingredient identified "
+            + "by the index number used in the displayed ingredient list. "
+            + "Input value will be added to existing value.\n"
+            + "Parameters: "
+            + PREFIX_INDEX + "INDEX "
+            + PREFIX_INGREDIENT_QUANTITY + "QUANTITY "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_INDEX + "1 "
+            + PREFIX_INGREDIENT_QUANTITY + "10 ";
+
+    public static final String MESSAGE_SUCCESS = "Depleted Ingredient: %1$s";
+    public static final String MESSAGE_DUPLICATE = "Ingredient is already in list";
```

Comment:
I don't think there will ever be duplicates (it doesn't make sense)
However, you need a different kind of error message- if I deplete an amount greater than what I currently have, an error message should be thrown.